recipes/datamountaineer-schemaregistry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: datamountaineer-schemaregistry
+  version: "0.2"
+
+source:
+  git_url: https://github.com/datamountaineer/python-serializers.git
+  git_rev: 3a55967d29524f406548f2afa844f92f18d12a7d
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - fastavro
+    - python-avro
+
+  run:
+    - python
+    - fastavro
+    - python-avro
+
+test:
+  requires:
+    - mock
+    - pytest
+  commands:
+    - py.test {{ environ['SRC_DIR'] }}
+  imports:
+    - datamountaineer
+    - datamountaineer.schemaregistry
+    - datamountaineer.schemaregistry.tests
+    - datamountaineer.schemaregistry.client
+    - datamountaineer.schemaregistry.serializers
+
+about:
+  home: https://github.com/datamountaineer/python-serializers
+  license: Apache 2.0
+  summary: 'DataMountaineer Python 3.5 Confluent Schema Registry Client'
```

Comment:
It says Python 3.5 but passed 3.4. Is that OK or should we change to `skip: True  # [not py35]`?